eml/screens/lectures/ReactSliderProgress.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect, StyleSheet } from 'react';
+import { View, Text } from 'react-native';
+import Slider from '@react-native-community/slider';
+import tailwindConfig from '../../tailwind.config';
+
+
+
+const ReactSliderProgress = ({ elapsedMs = 10000, totalMs = 20000, color, height, videoRef }) => {
+
+    const convertMsToTime = (ms) => {
+        let seconds = Math.floor((ms / 1000) % 60);
+        let minutes = Math.floor((ms / (1000 * 60)) % 60);
+
+        seconds = seconds < 10 ? '0' + seconds : seconds;
+        minutes = minutes < 10 ? '0' + minutes : minutes;
+
+        return `${minutes}:${seconds}`;
+    }
```

Comment:
**Suggestion: Moving this function to `services/utilityFunction.js`**

Consider moving this utility function into `services/utilityFunction.js`. That way I will gladly create a unit test for this function for you 😃

Reply:
Seems a bit redundant because we won't use it anywhere else, but we can do it if you like